tests/python/gaia-ui-tests/gaiatest/apps/system/app.py

```diff
@@ -23,25 +26,38 @@ class System(Base):
     _software_home_button_locator = (By.ID, 'software-home-button')
 
     def wait_for_status_bar_displayed(self):
-        self.wait_for_element_displayed(*self._status_bar_locator)
+        element = Wait(self.marionette).until(
+            expected.element_present(*self._status_bar_locator))
+        Wait(self.marionette).until(expected.element_displayed(element))
```

Comment:
`element_displayed` can now take a locator, so this can be written as:

```python
Wait(self.marionette).until(expected.element_displayed(*self._status_bar_locator)
```